sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies_async.py

```diff
@@ -62,7 +63,10 @@ async def send(self, request):
             request.context.options.pop('raw_response_hook', self._response_callback)
 
         response = await self.next.send(request)
-        await response.http_response.load_body()
+        try:
+            await response.http_response.load_body()
+        except ServerTimeoutError:
```

Comment:
This is temporary (since I know we can't take a dep. out on `ServerTimeoutError`). Instead of importing, could I just inspect the error and look at it's message? I want to make sure this except only captures what we are expecting, or is the only case we fail on `load_body` we will want to re-raise as `IncompleteReadError`?